_faqs/automated_link_clicking.md

```diff
@@ -0,0 +1,14 @@
+---
+question: Some links (URLs) in emails being sent through SparkPost are being visited before email is delivered
+author: Chris Wilson
+category: Platform
+---
+
+
+SparkPost has builtin protection from potential malicious users that use link redirectors and shorteners to hide prohibited content. SparkPost aggressively monitors for prohibited content to protect not only it's systems but all of the legitimate users on the platform. 
+
+To help prevent these types of issues, while customers are building their reputation on the SparkPost platform it will follow some links to ensure the sender is not doing bad things. 
+
+SparkPost tries to balance those algorithms to be aggressive enough to handle the bad users actively trying to circumvent the system while not being burdensome to legitimate users.
+
+You can learn more about SparkPost's messaging policy [here](https://www.sparkpost.com/policies)
```

Comment:
I tend not to put periods one a sentence that ends with a link for fear it will mess up the link. I will add the period.